src/web/pages/groups/Header.tsx

```diff
@@ -7,25 +7,35 @@ import React from 'react';
 import TableHead from 'web/components/table/TableHead';
 import TableHeader from 'web/components/table/TableHeader';
 import TableRow from 'web/components/table/TableRow';
-import withEntitiesHeader from 'web/entities/withEntitiesHeader';
+import withEntitiesHeader, {
+  ActionsColumn,
+} from 'web/entities/withEntitiesHeader';
 import useTranslation from 'web/hooks/useTranslation';
-import PropTypes from 'web/utils/PropTypes';
+import {SortDirectionType} from 'web/utils/SortDirection';
+
+export interface HeaderProps {
+  actionsColumn?: ActionsColumn;
+  sort?: boolean;
+  currentSortBy?: string;
+  currentSortDir?: SortDirectionType;
+  onSortChange?: (sortBy: string) => void;
+}
 
 const Header = ({
   actionsColumn,
   sort = true,
   currentSortBy,
   currentSortDir,
   onSortChange,
-}) => {
+}: HeaderProps) => {
   const [_] = useTranslation();
   return (
     <TableHeader>
       <TableRow>
         <TableHead
           currentSortBy={currentSortBy}
           currentSortDir={currentSortDir}
-          sortBy={sort ? 'name' : false}
+          sortBy={sort ? 'name' : undefined}
```

Comment:
It seems TableHead got extended for this use case some time ago.
```suggestion
          sort={sort}
          sortBy="name"
```